src/sentry/integrations/bitbucket/issues.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import absolute_import
+from sentry.integrations.issues import IssueBasicMixin
+from sentry.integrations.exceptions import ApiError, IntegrationError
+
+
+ISSUE_TYPES = (
+    ('bug', 'Bug'), ('enhancement', 'Enhancement'), ('proposal', 'Proposal'), ('task', 'Task'),
+)
+
+PRIORITIES = (
+    ('trivial', 'Trivial', ), ('minor', 'Minor', ), ('major', 'Major'), ('critical', 'Critical'),
+    ('blocker', 'Blocker'),
+)
+
+ERR_404 = (
+    'Bitbucket returned a 404. Please make sure that '
+    'the repo exists, you have access to it, and it has '
+    'issue tracking enabled.'
+)
+
+
+class BitbucketIssueSyncMixin(IssueBasicMixin):
```

Comment:
I would maybe label this as `BitbucketIssueBasicMixin` or something since we aren't actually doing the issue sync for bb yet.